rest-api-spec/src/yamlRestTest/resources/rest-api-spec/test/tsdb/40_search.yml

```diff
@@ -189,8 +189,8 @@ fetch a tag:
 ---
 aggregate a dimension:
   - skip:
-      version: " - 7.99.99"
```

Comment:
I am not sure why this test would not work in 8.0.

Reply:
I think it's an old code, and I update it.